cmd/util.go

```diff
@@ -17,3 +19,29 @@ func FatalOnError(err error) {
 		log.Fatal(err)
 	}
 }
+
+// GetHome is an helper function to get current home directory
+func GetHome() string {
+	usr, _ := user.Current()
+	return usr.HomeDir
+}
+
+// Dump is handy dumps of structure as json (almost any structures)
+func Dump(cls interface{}) {
+	data, err := json.MarshalIndent(cls, "", "    ")
```

Comment:
Maybe we can replace it with 
```
fmt.Println(Sdump(cli))
```
Since I suppose this is a debug utils, I'm not sure we should keep it in our codebase :)

Reply:
Yeah, this is debug tools, so can be removed before merge. :)